Cubical/Modalities/Modality.agda

```diff
@@ -58,15 +58,14 @@ record Modality ℓ : Type (ℓ-suc ℓ) where
   ◯-preservesEquiv : {A B : Type ℓ} (f : A → B) → isEquiv f → isEquiv (◯-map f)
   ◯-preservesEquiv f f-ise = isoToIsEquiv (iso _ (◯-map inv) to-from from-to) where
     open Iso (equivToIso (f , f-ise))
-    abstract
-      to-from : ∀ ◯b → ◯-map f (◯-map inv ◯b) ≡ ◯b
-      to-from = ◯-elim
-        (λ ◯b → ◯-=-isModal (◯-map f (◯-map inv ◯b)) ◯b)
-        (λ b → cong (◯-map f) (◯-map-β inv b) ∙ ◯-map-β f (inv b) ∙ cong η (rightInv b))
-      from-to : ∀ ◯a → ◯-map inv (◯-map f ◯a) ≡ ◯a
-      from-to = ◯-elim
-          (λ ◯a → ◯-=-isModal (◯-map inv (◯-map f ◯a)) ◯a)
-          (λ a → cong (◯-map inv) (◯-map-β f a) ∙ ◯-map-β inv (f a) ∙ cong η (leftInv a))
+    to-from : ∀ ◯b → ◯-map f (◯-map inv ◯b) ≡ ◯b
```

Comment:
I wonder why these things were abstract before? Efficiency?

Reply:
I'm not sure, with all of them removed it is slightly slower, but not to an extent that is unreasonable for files in this library (e.g. it's faster than `GroupoidLaws`). For some reason I was getting an error when compiling this file with `Σ-isModal` marked abstract, so first I removed just that instance of 'abstract', but then after testing just removed them all.